operator/src/main/resources/Operator.properties

```diff
@@ -205,7 +205,8 @@ WLSDO-0012=Managed Server ''{0}'' specified in the domain resource does not exis
 WLSDO-0013=Istio is enabled and the domain resource specified to expose channel {0} in the adminServices section. The channel name default, \
 default-admin, default-secure are internal to the WebLogic Kubernetes Operator. Please create the network channel with a different name in the WebLogic Domain before \
 exposing it.
-WLSDO-0014="The introspect job had failed, retrying introspection job. Current failure count: {0}."
+WLSDO-0014="The domain {0} introspect job failed, it will automatically retry in {1} seconds."
+WLSDO-0015="Retrying failed domain {0} introspect job {1} of {2}."
```

Comment:
(1) Suggested new text:  Retrying failed domain {0} introspect job, retry {1} of a maximum of {2}.

(2) Can you add a new 'Retrying' INFO for the 'infinite' retry case?

Reply:
1 is done. 
2 has a severe message 

WLSKO-0157=Domain {0} is not valid: {1}